src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -372,7 +372,7 @@ public function setRequiredFieldsAction(Request $request)
     /**
      * Search for customers by query.
      *
-     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))")
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller')) || is_granted('create', 'AdminOrders')")
```

Comment:
Why do you mix permissions between Orders and Customers?

Reply:
Because this action is used in Orders page (and might even be used elsewhere in future), but it does still belong to CustomerController.

Reply:
Ok I get it, but it still seems dangerous to me to mix these permissions Although it seems logical to have both permissions when you need to create an order It's not up to the annotation to manage this logic, or else we are gonna hide permission logic in weird places and won't understand why some weird behaviour are happening

Reply:
This has to be discussed with Product Managers however it can be solved after QA check 👍